Lib/test/test_posix.py

```diff
@@ -1991,6 +1991,12 @@ def test_setscheduler_only_param(self):
     @requires_sched
     @unittest.skipIf(sys.platform.startswith(('freebsd', 'netbsd')),
                      "bpo-34685: test can fail on BSD")
+    @unittest.skipIf(platform.libc_ver()[0] == 'glibc' and
+                     os.sched_getscheduler(0) in [
+                        os.SCHED_BATCH,
+                        os.SCHED_IDLE,
+                        os.SCHED_DEADLINE],
```

Comment:
`SCHED_DEADLINE` [was added in 3.14](https://github.com/python/cpython/issues/127688).
```suggestion
                        os.SCHED_IDLE],
```